src/plugins/dash/videojs-dash.js

```diff
@@ -94,20 +87,92 @@ class Html5DashJS {
     // - mediaKeyMessageError (only fires under 'might not work' circumstances)
     // eslint-disable-next-line complexity
     this.retriggerError_ = (event) => {
-      if (event.type === 'error') {
-
-        /* initialize errros for ref https://cdn.dashjs.org/latest/jsdoc/core_errors_Errors.js.html
-          map to general init error
-        */
-        if (event.error.code >= 10 && event.error.code <= 35) {
-          this.player.error({ code: 4, dashjsErrorCode: event.error.code, message: event.error.message });
-        } else if (event.error.code < 10) { // network errors
-          this.player.error({ code: event.error.code, dashjsErrorCode: event.error.code, message: event.error.message });
-        } else if (event.error.code >= 100 && event.error.code <= 114) { // Protection Errors  https://cdn.dashjs.org/latest/jsdoc/streaming_protection_errors_ProtectionErrors.js.html
-          this.player.error({ code: 5, dashjsErrorCode: event.error.code, message: event.error.message });
```

Comment:
@tsi 
This part consolidated the dash errors to more general videojs errors so I think we should keep it

Reply:
@yaniv-cld pls have a look at merging this together, I don't want to break any error handling we had here.